cluster/manifest/load.go

```diff
@@ -13,37 +13,52 @@ import (
 	manifestpb "github.com/obolnetwork/charon/cluster/manifestpb/v1"
 )
 
-// Load loads a cluster from disk and returns true if cluster was loaded from a legacy lock file.
-// It supports reading from both cluster manifest and legacy lock files.
+// LoadManifest loads a cluster manifest from disk by reading either from cluster manifest or legacy lock file.
 // If both files are provided, it first reads the manifest file before reading the legacy lock file.
-func Load(manifestFile, legacyLockFile string, lockCallback func(cluster.Lock) error) (*manifestpb.Cluster, error) {
+func LoadManifest(manifestFile, legacyLockFile string, lockCallback func(cluster.Lock) error) (*manifestpb.Cluster, error) {
```

Comment:
rename `LoadCluster`, lets use accurate names, "LoadCluster returns the current cluster state by reading..."